pvr.nextpvr/changelog.txt

```diff
@@ -1,5 +1,8 @@
 v21.0.4
 - Allow control of recording and timers access
+- Strings.po was missing some hardcoded settings values
```

Comment:
The changelog is for users really. So this is not infomation they really need. It's no harm to have it but not required.